pkg/consensus/consensus_test.go

```diff
@@ -34,18 +31,18 @@ func TestNewService(t *testing.T) {
 func TestService_GetVerified(t *testing.T) {
 	srv := newTestService(t)
 	txs := []*transaction.Transaction{
-		newMinerTx(1),
-		newMinerTx(2),
-		newMinerTx(3),
-		newMinerTx(4),
+		transaction.NewMinerTX(),
+		transaction.NewMinerTX(),
+		transaction.NewMinerTX(),
+		transaction.NewMinerTX(),
```

Comment:
Shouldn't it be `NewMinerTXWithNonce()`?

Reply:
And everywhere else in this file. Random nonce might lead to random test results here if there to occur some duplicate nonces.